server/src/graphql/resolvers/mutations.js

```diff
@@ -15,4 +15,17 @@ module.exports = {
     const user = await User.create({ id: uuidv4(), email, password: hash });
     return user ? true : false;
   },
+
+  deleteRig: async (_, { id }) => {
+    const isDeleted = await Rig.destroy({ where: { id } });
+    return isDeleted ? true : false;
```

Comment:
make sure to keep the return value consistent between all delete mutations. Some of the other mutations return the entire row and other ones just the ID. I'd recommend returning the id if the deletion was successful and an error if something went wrong. Returning false to the FE isn't helpful when figuring out what went wrong.

Reply:
☝️